superset-frontend/packages/superset-ui-core/test/time-comparison/getTimeOffset.test.ts

```diff
@@ -0,0 +1,162 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { getTimeOffset, parseDttmToDate } from '@superset-ui/core';
+
+describe('parseDttmToDate', () => {
```

Comment:
Can you avoid `describe` and just use `test` to follow this [best practice](https://github.com/apache/superset/wiki/Testing-Guidelines-and-Best-Practices#avoid-nesting-when-youre-testing)? We're slowly refactoring the tests to move in that direction. 

I noticed that this file name is `getTimeOffset.test.ts` but it's also testing `parseDttmToDate`. Can you rename the file to something more generic or breaking it into 2 test files?

Reply:
Done!